django/contrib/admin/options.py

```diff
@@ -1194,21 +1185,60 @@ def response_change(self, request, obj):
             self.message_user(request, msg, messages.SUCCESS)
             return self.response_post_save_change(request, obj)
 
+    def admin_viewonsite_url(self, obj):
```

Comment:
It seems like it would be more consistent if the methods were prefixed with `get_` as `get_view_on_site_url` is. Without the prefix, I'd expect a name like `admin_index_url` to be a property.


I don't see the usefulness of `admin_viewonsite_url` as a layer of indirection to `get_view_on_site_url()`.